engine/src/main/java/org/hibernate/search/bridge/spi/FieldType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge.spi;
+
+/**
+ * The type of an indexed field.
+ *
+ * @author Gunnar Morling
+ * @hsearch.experimental This contract is currently under active development and may be altered in future releases,
+ * breaking existing users.
+ */
+public enum FieldType {
```

Comment:
Ha, thank you now I understand. Makes sense, and +1 to shorten their names. 

Wondering if we could (not now but in a future iteration) essentially have the builder provide the "adder" hook to the user, and replace the LuceneOptions as only way to add things to the "document".